ktcacheorbust.cc

```diff
@@ -154,23 +184,27 @@ bool CacheOrBust::Worker::do_get(
 
   if (tokens.size() < 2)
     return sess->printf("CLIENT_ERROR missing key\r\n");
-  if (tokens.size() < 3)
```

Comment:
is this not still a case to handle? or does the actual test for it being correct happen later?

Reply:
It happens later

Reply:
👍